trin-core/src/cli.rs

```diff
@@ -1,4 +1,4 @@
-use crate::portalnet::types::HexData;
+use crate::portalnet::types::messages::HexData;
```

Comment:
A good practice is to keep strong separation between organizational changes and functionality changes.  A good way to deal with this is to extract just the part that moved `HexData` to this new location and then open a pull request with just that change.  Then you can rebase this branch on top of that other PR which will allow a reviewer of this PR to not have to sort through more changed lines than absolutely necessary.